src/desktop/space/popup.rs

```diff
@@ -70,6 +74,7 @@ impl LayerSurface {
                         RenderPopup {
                             location: offset,
                             popup,
+                            parent_layer: RenderZindex::Overlay,
```

Comment:
This should only be the case, if this `LayerSurface::layer` actually returns `Overlay`, all other popups should be on the same `Popups` layer.

Reply:
Changed a lite bit the logic, now z_index is resolved when RenderPopup is created, that made it a lite bit easier to get the logic right for Windows and LayerSurface separately